### PR TITLE
Cancellation charge changes

### PR DESCRIPTION
**Pull request by Sriep**
0chain: https://github.com/0chain/0chain/pull/1614
gosdk: https://github.com/0chain/gosdk/pull/554
zboxcli: https://github.com/0chain/zboxcli/pull/290

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | cancellation_charge  |
| gosdk  | allocation_min_lock  |
| zboxcli  | allocation_min_lock  |
| zwalletcli  | staging  |
| 0chain  | cancellation_charge  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | cancellation_charge  |
| gosdk  | allocation_min_lock  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | cancellation_charge  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | cancellation_charge  |
| gosdk  | allocation_min_lock  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | cancellation_charge  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/api_tests/endpoints_test.go [hunk 1]
```diff
@@ -53,7 +54,7 @@ func v1ScrestAllocation(t *testing.T, clientId string) (*model.Allocation, *rest
 func v1ScrestAllocBlobbers(t *testing.T, allocationData string) (*[]string, *resty.Response, error) { //nolint
 	var blobbers *[]string
 
-	httpResponse, httpError := zeroChain.GetFromSharders(t, "/v1/screst/"+STORAGE_SMART_CONTRACT_ADDRESS+"/alloc_blobbers?allocation_data="+allocationData, &blobbers)
+	httpResponse, httpError := zeroChain.GetFromSharders(t, "/v1/screst/"+STORAGE_SMART_CONTRACT_ADDRESS+"/alloc-blobbers?allocation_data="+allocationData, &blobbers)
```
Comment (dabasov) [inline tests/api_tests/endpoints_test.go:57]: I haven't seen endpoint name change in 0chain

**Review by dabasov — CHANGES_REQUESTED** [review]

File: tests/api_tests/endpoints_test.go (hunk 1 above)
Comment (dabasov) [reply tests/api_tests/endpoints_test.go:57]: do not suggest to change name, as there are a lot of clients who use it and coordination of change an be painful

**Comment by Sriep** [comment]
not needed

**Pull request closed without merging**